yt/frontends/amrvac/data_structures.py

```diff
@@ -191,6 +208,17 @@ def __init__(
         c_adiab = None
         e_is_internal = None
         if parfiles is not None:
+            parfiles = list(always_iterable(parfiles))
+            ppf = Path(parfiles[0])
+            if not ppf.is_absolute() and Path(filename).resolve().is_relative_to(
+                ytcfg["yt", "test_data_dir"]
```

Comment:
Should this also be `.resolve()`d?

Reply:
what do you mean ?

Reply:
@cphyc I think you meant `ytcfg["yt", "test_data_dir"]` ? If so, I'm always assuming this path is absolute, but I realise now that it's a bit naive and would probably not work great with local config files attached to a project. I'll give it a bit more thoughts.

Reply:
Nvm!